Check the tasks (#4)
jmredfern/vscode-sort-selection: sort-selection/README.md

# Sort a delimited list

> This plugin adds the "Sort with a punctation mark" command.

Reviewer: ```suggestion
This plugin adds the "Sort delimited list" command.
```

Also please update the name in the code so that the command name matches the project name
